geocoding/location-service/src/app/main.py

```diff
@@ -34,9 +35,11 @@ def clear_fake_geocoder():
     mongo_client = None
     if 'DB_CONNECTION_STRING' in environ:
         mongo_client = pymongo.MongoClient(environ['DB_CONNECTION_STRING'])
-
-    admins_fetcher = AdminsFetcher(access_token, mongo_client)
-    mapbox_geocoder = Geocoder(access_token, admins_fetcher)
+    rate_limit = 600 # requests per minute, default
+    if 'MAPBOX_GEOCODE_RATE_LIMIT_PER_MIN' in environ:
+        rate_limit = int(environ['MAPBOX_GEOCODE_RATE_LIMIT_PER_MIN'])
```

Comment:
this can be clubbed down to `rate_limit = int(environ.get('MAPBOX_GEOCODE_RATE_LIMIT_PER_MIN', 600))`